### ISSUE TITLE
Cannot save alt text

### ISSUE THREAD

**Issue by ericswpark** [mastodon/mastodon-android#771]
I was writing alt text for an image today, and realized that there was no "Save" button or anything when I went to save it. So I pressed the back arrow at the top left of the screen and it deleted all of the alt text that I just wrote. 

This was from the share modal, sharing an image directly to Mastodon.

**Comment by ericswpark** [comment]
@grishka it certainly was, because when I clicked on the post icon it warned me that the image had no alt text.

**Comment by grishka** [comment]
Then I'm unable to reproduce this.

**Comment by ericswpark** [comment]
For some reason I'm unable to reproduce it anymore either. Seems to be user error. Sorry! Closing the issue.
